### ISSUE TITLE
Calendar(iOS): Events longer than 1 day are drawn like they are one day event

### ISSUE THREAD

**Issue by Natalia-Hristova** [telerik/nativescript-ui-feedback#373]
### Tell us about the problem
Longer events are shown like they end in the same day (when the event is ~ 3 days long)

### Which platform(s) does your issue occur on?
iOS

### Please provide the following version numbers that your issue occurs with:
- **Progress NativeScript UI version:** 3.2.0
- **CLI:** 3.3.0
- **Cross-platform modules:** 3.3.0
- **Runtime(s):** 3.3.0

### Please tell us how to recreate the issue in as much detail as possible.
1. In SDK example modify [events-view-modes-model.ts](https://github.com/telerik/nativescript-ui-samples/blob/release/sdk/app/calendar/events-view-modes/events-view-modes-model.ts) file so there are multi-day event(s)
2. Run SDK -> Calendar -> Events view modes example and go to the date with the longer event

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.
e.g. update events-view-modes-model.ts file
```TypeScript
...
    calendarEvents: Array<calendarModule.CalendarEvent>;
    private _viewModesInfo;
    private _eventTitles: Array<string> = ["Meeting with Jack", "Lunch with Peter", "Planning meeting",
        "Go shopping", "Very important meeting", "Another meeting"];
    private _eventColors: Array<Color> = [new Color("#0288D1"), new Color("#009688"), new Color("#E040FB")];

     
    constructor() {
        super();
        this.createEvents();
        this._viewModesInfo = {
            options: ["None", "Inline", "Popover (available on iPad only)"],
            index: 0
        };
    }
    
    get eventsMode() {
        return this.get("eventsViewMode");
    }
    
    set eventsMode(value: string) { 
        this.set("eventsViewMode", value);
    }
    
    public createEvents() {
    var events: Array<calendarModule.CalendarEvent> = new Array<calendarModule.CalendarEvent>();
        var now: Date = new Date();
        var startDate: Date = new Date(now.getFullYear(), now.getMonth(), now.getDate());
        var endDate: Date = new Date(now.getFullYear(), now.getMonth(), now.getDate());

        startDate.setHours(9);
        // endDate.setHours(10);
        var event = new calendarModule.CalendarEvent(this._eventTitles[0], new Date(2017,9,30,11), new Date(2017,10,1,8), false, this._eventColors[0]);
        events.push(event);

        startDate.setHours(12);
        endDate.setHours(13);
        var event = new calendarModule.CalendarEvent(this._eventTitles[1], startDate, endDate, false, this._eventColors[1]);
        events.push(event);

        startDate.setHours(13);
        endDate.setHours(14);
        var event = new calendarModule.CalendarEvent(this._eventTitles[2], startDate, endDate, false, this._eventColors[0]);
        events.push(event);

        startDate.setHours(20);
        endDate.setHours(22);
        var event = new calendarModule.CalendarEvent(this._eventTitles[3], startDate, endDate, false, this._eventColors[2]);
        events.push(event);

        startDate.setHours(2);
        endDate.setHours(4);
        var event = new calendarModule.CalendarEvent(this._eventTitles[4], startDate, endDate, false, this._eventColors[0]);
        events.push(event);

        startDate.setHours(16);
        endDate.setHours(17);
        var event = new calendarModule.CalendarEvent(this._eventTitles[5], startDate, endDate, false, this._eventColors[0]);
        events.push(event);

        this.calendarEvents = events;
    }
...
```
![2017-10-30](https://user-images.githubusercontent.com/5803037/32188966-fbbdf322-bdb1-11e7-8971-ce11b5e48089.png)

**Comment by paidTrainee27** [comment]
Have You tried using  **selectionMode** property.
[here](http://docs.telerik.com/devtools/nativescript-ui/Controls/NativeScript/Calendar/selection-modes) is the ref.

**Comment by zbranzov** [comment]
This issue should be fixed in nativescript-ui-calendar@3.8.1
